Sources/Abstract/Free/Multiset+FreeCommutativeMonoid.swift

```diff
@@ -0,0 +1,168 @@
+#if SWIFT_PACKAGE
+	import Operadics
+#endif
+
+public struct Multiset<A>: Equatable where A: Hashable {
+	private var storage: [A : Int] = [:]
+	public private(set) var count = 0
+
+	public init() {}
+
+	public init <C> (_ collection: C) where C: Collection, C.Element == A {
```

Comment:
This bound can be relaxed to `Sequence`

Reply:
The reason for this constraint is that I want to avoid the possibility of an infinite `Sequence`.

Reply:
@CodaFi I relaxed the constraint. I'll merge as soon as you approve.